Added some icons I've made and updated the text file.

Imernium opened the pull request.

elrumo commented:
Thanks, they look great but could you please add the low res (256px) and high res .pngs please? Or else the preview won't display on the website. Thanks!

Imernium commented:
> Thanks, they look great but could you please add the low res (256px) and high res .pngs please? Or else the preview won't display on the website. Thanks!

Sure! Just added some additional icons, updated the text file, and added high and low res .pngs to the commit. I'm new to all this so let me know if you need anything else added or corrected.

elrumo commented:
Thanks! It was all good now. I'm actually building a new system that will allow any one to submit the icons as pngs directly form macosicons.com so people who don't know how to use GitHub will be able to contribute too :)
